test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2996,3 +3003,9 @@ UCS_TEST_P(test_ucp_sockaddr_protocols_err_sender,
 }
 
 UCP_INSTANTIATE_CM_TEST_CASE(test_ucp_sockaddr_protocols_err_sender)
+UCP_INSTANTIATE_TEST_CASE_TLS_GPU_AWARE(test_ucp_sockaddr_protocols_err_sender,
+                                        dc_mlx5, "dc_mlx5,ud")
+UCP_INSTANTIATE_TEST_CASE_TLS_GPU_AWARE(test_ucp_sockaddr_protocols_err_sender,
+                                        rc_mlx5, "rc_mlx5")
+UCP_INSTANTIATE_TEST_CASE_TLS_GPU_AWARE(test_ucp_sockaddr_protocols_err_sender,
+                                        rc_verbs, "rc_verbs")
```

Comment:
do we really need to add all of these? it increases testing time

Reply:
IMO, we need to test RC withoud UD keepalive.
combined `rc_mlx5` and `rc_verbs` to `rc_no_ud`